default.nix

```diff
@@ -20,8 +20,6 @@ let
     coreutils
     git
     gitAndTools.hub
-    nix-prefetch-git
```

Comment:
This should stay.

Reply:
Oh I thought it was also bundled as port of a nix installation

Reply:
Wait till you realize Hydra ships with its own copy of nix-prefetch-git from like 2012...